python/cuml/cuml/testing/utils.py

```diff
@@ -650,22 +649,6 @@ def compare_svm(
     assert intersection_len > average_len / 8
 
 
-def compare_probabilistic_svm(
```

Comment:
This utility was only called once in one test file. `pytest` doesn't rewrite assert statements outside of test files, so having this utility here meant that if something did fail you'd only get an `AssertionError` rather than a nice error with details.

I've moved an updated version of this inline to the test in question, and updated it to better work with multiclass support.